mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule AssertValue.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [
-      applications: [],
+      applications: [:iex],
```

Comment:
We could also do `extra_applications: [:iex]`here but I'm not sure which version of elixir started supporting this keyword.